webapp/public/js/domjudge.js

```diff
@@ -425,6 +425,15 @@ function processAjaxResponse(jqXHR, data) {
     if (jqXHR.getResponseHeader('X-Login-Page')) {
         window.location = jqXHR.getResponseHeader('X-Login-Page');
     } else {
+        var newCurrentContest = parseInt(jqXHR.getResponseHeader('X-Current-Contest'));
+        var currentContest = parseInt($('[data-current-contest]').data('current-contest'));
+
+        // If the contest ID changed from another tab, reload or whole page
+        if (newCurrentContest !== currentContest) {
+            window.location.reload();
```

Comment:
We use `location.reload()` in the same file, should probably be consistent either way.

Reply:
Seems we use window.location way more often so updated the one place we didn't to use it.